week-7/Homework/mandatory/1-debugging-practice/script.js

```diff
@@ -38,7 +38,7 @@ function submit() {
     return false;
   } else {
     let book = new Book(title.value, title.value, pages.value, check.checked);
```

Comment:
There is a missed bit here. Can you spot what that is? Pinch of a hint: there is a missing parameter and one of the value is passed twice instead of the missed one.